LPS-52078

jonmak08 opened the pull request:
Hey @natecavanaugh,

Attached is an update for http://issues.liferay.com/browse/LPS-52078. 

Please let me know if you have any questions.

Thanks!

westonhancock commented:
@natecavanaugh,

If this get push on up to Brian, "initalEditor" should be changed to "initialEditor".

jonmak08 commented:
Hey @natecavanaugh and @westonhancock,

I have fixed the typo and updated the branch on this pull request.  The change can be found at https://github.com/jonmak08/liferay-portal/commit/cca10c780bf2ad5471437a17cd9ca4e486d581fc.

Please let me know if you have any questions.  Thanks!

natecavanaugh commented:
Merged and pushed to Brian. Thanks :)
Pull request submitted at: https://github.com/brianchandotcom/liferay-portal/pull/24474

View just my changes: https://github.com/natecavanaugh/liferay-portal/compare/jonmak08:0f844dae04...8642e7d21c
View total diff: https://github.com/natecavanaugh/liferay-portal/compare/3f9c995e3f...8642e7d21c
